### PR TITLE
(maint) Fix yum repo path for shipping

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: ext/build_defaults.yaml [hunk 1]
```diff
@@ -21,7 +21,7 @@ final_mocks: 'pl-el-5-x86_64 pl-el-5-i386 pl-el-6-x86_64 pl-el-6-i386 pl-el-7-x8
 # The host that contains the yum repository to ship to
 yum_host: 'yum.puppetlabs.com'
 # The remote path the repository on the yum\_host
-yum_repo_path: '/some/repo/'
+yum_repo_path: '/opt/repository/yum/'
```
Comment (haus) [inline ext/build_defaults.yaml]: does the trailing slash matter? it is not present for the apt_path

Comment (melissa) [inline ext/build_defaults.yaml]: That's how it is for all other projects, and that's what I used for the 0.3.0 ship

**Comment by haus** [comment]
:racehorse: :+1:

**Comment by melissa** [comment]
That fail was not my fault, it timed out

**Pull request merged**